### ISSUE TITLE
The alt text should describe the purpose of the link

### ISSUE THREAD

**Issue by JacquelineOmollo** [unitymakesus/justgetsimple#67]
<a href="https://justgetsimple.com/ncfragilex/resources/" data-wpel-link="internal"><img loading="lazy" class="size...</a> 1.1.1
About, Resources and Donate have missing alt text. The Image should have one link instead of two.
